etc/conf/atTelemetry.yaml

```diff
@@ -1,5 +1,8 @@
 # Configuration file for the AuxTelescope
 
+# logging level
```

Comment:
Can you add `Default` here so that it reads `Default logging level`?

Reply:
Yes. Done.